kubernetes/loculus/templates/loculus-database-standin.yaml

```diff
@@ -39,5 +39,5 @@ spec:
         - name: POSTGRES_HOST_AUTH_METHOD
           value: "trust"
         - name: LOCULUS_VERSION
-          value: {{ $dockerTag }}
+          value: "commit-e3f268f"
```

Comment:
For info - this isn't consumed at all by Postgres so it could also just be removed

Reply:
(It is used specifically to cause the db to be recreated because that is useful in some scenarios)